app/models/post.rb

```diff
@@ -2,4 +2,14 @@ class Post < ApplicationRecord
   belongs_to :user
```

Comment:
- [ ]  the way that you used it may be redundant in a large code base so it would be better to use `belongs_to :author, class_name: 'User'` instead of `belongs_to :user`